sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -567,74 +266,136 @@ export class TopicClient implements Client {
-    _response: HttpOperationResponse;
+// @public (undocumented)
+export interface Track2PeekedMessage extends ReceivedMessageInfo {
+}
+
+// @public (undocumented)
+export interface Track2PlainContext {
+}
+
+// @public (undocumented)
+export class Track2QueueConsumerClient {
+    constructor(connectionString: string, queueName: string, options?: Track2QueueConsumerClientOptions);
+    // (undocumented)
+    close(): Promise<void>;
+    // (undocumented)
+    consume(sessionId: string, mode: "PeekLock", handlers: Track2ReceiverHandlers<Track2SessionMessage, Track2SessionContext & Track2SettleableContext>): Track2CloseableThing;
+    // (undocumented)
+    consume(sessionId: string, mode: "ReceiveAndDelete", handlers: Track2ReceiverHandlers<Track2SessionMessage, Track2SessionContext>): Track2CloseableThing;
+    // (undocumented)
+    consume(mode: "PeekLock", handlers: Track2ReceiverHandlers<Track2Message, Track2SettleableContext>): Track2CloseableThing;
+    // (undocumented)
+    consume(mode: "ReceiveAndDelete", handlers: Track2ReceiverHandlers<Track2Message, Track2PlainContext>): Track2CloseableThing;
+    // (undocumented)
+    fetch(sessionId: string, mode: "PeekLock", options?: Track2FetchOptions): Track2FetchResult<Track2Message, Track2SettleableContext>;
+    // (undocumented)
+    fetch(sessionId: string, mode: "ReceiveAndDelete", options?: Track2FetchOptions): Track2FetchResult<Track2Message, Track2PlainContext>;
+    // (undocumented)
+    fetch(mode: "PeekLock", options?: Track2FetchOptions): Track2FetchResult<Track2Message, Track2SettleableContext>;
+    // (undocumented)
+    fetch(mode: "ReceiveAndDelete", options?: Track2FetchOptions): Track2FetchResult<Track2Message, Track2PlainContext>;
+    // (undocumented)
+    peekWithoutLock(sessionId: string, messageCount?: number): Promise<Track2PeekedMessage[]>;
+    // Warning: (ae-forgotten-export) The symbol "Long" needs to be exported by the entry point index.d.ts
+    // 
+    // (undocumented)
+    peekWithoutLock(sessionId: string, fromSequenceNumber: Long_2, maxMessageCount?: number): Promise<Track2PeekedMessage[]>;
+    // (undocumented)
+    peekWithoutLock(messageCount?: number): Promise<Track2PeekedMessage[]>;
+    // (undocumented)
+    peekWithoutLock(fromSequenceNumber: Long_2, maxMessageCount?: number): Promise<Track2PeekedMessage[]>;
+    }
+
+// @public (undocumented)
+export interface Track2QueueConsumerClientOptions extends ServiceBusClientOptions {
+}
+
+// @public (undocumented)
+export class Track2QueueProducerClient {
+    constructor(connectionString: string, queueName: string, options?: ServiceBusClientOptions);
+    // (undocumented)
+    cancelScheduledMessage(sequenceNumber: Long_2): Promise<void>;
+    // (undocumented)
+    close(): Promise<void>;
+    // (undocumented)
+    schedule(scheduledEnqueueTimeUtc: Date, message: Track2SendableMessage): Promise<Long_2>;
+    // (undocumented)
+    send(message: Track2SendableMessage): Promise<void>;
```

Comment:
Do we need a send batch?

Reply:
We do. Currently only have the sendBatch which takes an enumerable - we'll need to code share and do something to what we do in EventHubs.

Reply:
Issue filed for this: #7673